src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -37,8 +37,9 @@
 						:key="`loan-${loan}`"
 						:loan-id="loan"
 						:lend-now-button="true"
-						custom-checkout-route="#show-basket"
```

Comment:
How are you handling showing checkout after the add to basket event now?

Reply:
These are the modifiers that set the hash to open the basket and I think they ensure the basket is "re-opened" when returning from login as well...

Reply:
We show it in the campaign progress bar instead and deactivate the loan card buttons.